### PR TITLE
add variable item `tidb_track_aggregate_memory_usage`

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5159
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -918,3 +918,13 @@ explain select * from t where age=5;
 - Default value: 0.8
 - TiDB triggers an alarm when the percentage of the memory it takes exceeds a certain threshold. For the detailed usage description of this feature, see [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-new-in-v409).
 - You can set the initial value of this variable by configuring [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-new-in-v409).
+
+### `tidb_track_aggregate_memory_usage` <!-- New in v5.0.0-rc -->
+
+> **Warning:**
+>
+> `tidb_track_aggregate_memory_usage` is currently an experimental feature. It is not recommended to use this feature in the production environment.
+>
```
Comment (yikeke) [inline system-variables.md:927]:
```suggestion

```

File: system-variables.md [hunk 2]
```diff
@@ -918,3 +918,13 @@ explain select * from t where age=5;
...
+
+### `tidb_track_aggregate_memory_usage` <!-- New in v5.0.0-rc -->
+
+> **Warning:**
+>
+> `tidb_track_aggregate_memory_usage` is currently an experimental feature. It is not recommended to use this feature in the production environment.
+> 
+- Scope: SESSION | GLOBAL
+- Default value: OFF
+- This variable indicates whether to track the memory usage of the aggregate function. When you enable this feature, it counts the memory usage of the aggregate function, which might cause the overall SQL memory statistics to exceed the threshold [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query), and then be affected by the behavior defined by [`oom-action`](/tidb-configuration-file.md#oom-action).
```
Comment (yikeke) [inline system-variables.md:930]:
```suggestion
- This variable controls whether to track the memory usage of the aggregate function. When you enable this feature, TiDB counts the memory usage of the aggregate function, which might cause the overall SQL memory statistics to exceed the threshold [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query), and then be affected by the behavior defined by [`oom-action`](/tidb-configuration-file.md#oom-action).
```

**Review by yikeke — COMMENTED** [review]
Rest LGTM

**Review by wshwsh12 — COMMENTED** [review]
LGTM

File: system-variables.md [hunk 3]
```diff
@@ -918,3 +918,13 @@ explain select * from t where age=5;
 - Default value: 0.8
 - TiDB triggers an alarm when the percentage of the memory it takes exceeds a certain threshold. For the detailed usage description of this feature, see [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-new-in-v409).
 - You can set the initial value of this variable by configuring [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-new-in-v409).
+
+### `tidb_track_aggregate_memory_usage` <!-- New in v5.0.0-rc -->
```
Comment (TomShawn) [inline system-variables.md:922]:
```suggestion
### `tidb_track_aggregate_memory_usage` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
```

File: system-variables.md [hunk 4]
```diff
@@ -918,3 +918,13 @@ explain select * from t where age=5;
 - Default value: 0.8
 - TiDB triggers an alarm when the percentage of the memory it takes exceeds a certain threshold. For the detailed usage description of this feature, see [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-new-in-v409).
 - You can set the initial value of this variable by configuring [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-new-in-v409).
+
+### `tidb_track_aggregate_memory_usage` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
```
Comment (TomShawn) [inline system-variables.md:922]:
```suggestion
### `tidb_track_aggregate_memory_usage` <span class="version-mark">New in v5.0.0-rc</span>
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**